assets/queries/ansible/aws/api_gateway_public_lambda/query.rego

```diff
@@ -0,0 +1,27 @@
+package Cx
+
+CxPolicy [  result ] {
+  document := input.document[i]
+  tasks := getTasks(document)
+  task := tasks[t]
+  awsApiGateway := task["community.aws.lambda"]
+  contains(awsApiGateway.state, "present")
+  role := awsApiGateway.role
+  endswith(role, "/*")
```

Comment:
I don't think the 'role' of a "community.aws.lambda" task concerns this query.
Instead, I think you should check if a "community.aws.lambda_policy" task gives permission to invoke a lambda function using public API Gateway. Check the implementation of the Terraform's version for this query done by Vasyl (the fields are similar).

Reply:
Hi Jorge. I don't know what happen but this query was not related al all indeed. I've changed everything: samples + query.